py/modbuiltins.c

```diff
@@ -431,7 +431,7 @@ STATIC mp_obj_t mp_builtin_repr(mp_obj_t o_in) {
 MP_DEFINE_CONST_FUN_OBJ_1(mp_builtin_repr_obj, mp_builtin_repr);
 
 STATIC mp_obj_t mp_builtin_round(mp_uint_t n_args, const mp_obj_t *args) {
-    // TODO really support second arg
+    // TODO second arg may lead to overflow
```

Comment:
I suggest adding "overflow" comment directly above the line where overflow may happen.